xbmc/AppParamParser.h

```diff
@@ -18,7 +18,7 @@ class CAppParamParser
 {
 public:
   CAppParamParser();
-  ~CAppParamParser();
+  ~CAppParamParser() = default;
```

Comment:
Inlining the dtor does not work here, if we don't want to include `CFileItemList` header, because of member `std::unique_ptr<CFileItemList> m_playlist;`

Causes a build breaker on Windows, works by luck only for the other platforms.